templates/sag-cc-builder-dev/README.md

```diff
@@ -16,12 +16,14 @@
      limitations under the License.                                                  
 
 -->
-# Command Central Product, Fix, and Asset Repositories
+# Internal Repositories for the Command Central Docker Builder
 
-Use this template to add product, fix, and asset repositories in Command Central.
+Use this template to add internal mirror repositories (located inside the Software AG network) to use with the Command Central Docker builder when building images for internal development or testing.
```

Comment:
Elly, maybe replace "internal mirror repositories" with
"internal product, fix and asset repositories", because we are losing information that asset repo is also created with the template.

Reply:
Okay, Yana, I updated the file.